lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -92,6 +92,10 @@ def execute(command, options = {})
       def shellescape(string)
         Shellwords.escape(string)
       end
+
+      def qpid_tools_installed?
```

Comment:
Any reason not to put it into the qpid feature? The `SystemHelpers` should contain only the more generic commands, usable across the definitions.